.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Please give the pull request a short but descriptive title.
+
+Use this template as a guide. Omit sections that don't apply. You may link to information rather than copy it.
+
+edX Internal Developers: Please also follow these steps to integrate your changes https://openedx.atlassian.net/wiki/spaces/RS/pages/1835106870/How+to+contribute+to+our+repositories
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
```

Comment:
"and why these changes were made"? 

🎱 Personally prefer a more active tone so that the audience feels compelled to take action. So more "what did you do and why did you do it" style wording.

Reply:
🎱 `Include implications for people using this change.` -> `How will these changes affect other [people|installations of edx|etc.]?`